src/imageproxy.rs

```diff
@@ -149,16 +150,10 @@ pub struct ImageProxyConfig {
     pub skopeo_cmd: Option<Command>,
 }
 
-impl ImageProxy {
-    /// Create an image proxy that fetches the target image, using default configuration.
-    pub async fn new() -> Result<Self> {
-        Self::new_with_config(Default::default()).await
-    }
+impl TryFrom<ImageProxyConfig> for Command {
+    type Error = anyhow::Error;
```

Comment:
Shouldn't libraries use `thiserror` instead?

Reply:
Mmm, I need to write a blog on that.

Reply:
https://users.rust-lang.org/t/why-i-use-anyhow-error-even-in-libraries/68592